components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -146,10 +162,15 @@ void RRTMGPRadiation::run_impl (const Real dt) {
         rei(i+1,k+1)   = d_rei(i,k);
         p_lev(i+1,k+1) = d_pint(i,k);
         t_lev(i+1,k+1) = d_tint(i,k);
-
-        Kokkos::parallel_for(Kokkos::ThreadVectorRange(team, m_ngas), [&] (const int& g) {
-          gas_vmr(i+1,k+1,g+1) = d_gas_vmr(i,k,g);
-        });
+        // gas concentrations
+        gas_vmr(i+1,k+1,1) = d_qv(i,k);
```

Comment:
What do you think about combining this with the setting of the GasConcs object below? The `gas_vmr` array is just a temporary, and existed only so that I could pass all the gases through the FM at once. Since we aren't doing that, there's really no reason to set things up like this here. The ultimate thing that gets passed into RRTMGP is a `GasConcs` object, with data populated with the gas volume mixing ratios. Seems like we could cut out a middle-man here. Also, I'm a little uncomfortable with assuming an ordering for the gases here, but maybe this is a lesser evil than doing string comparisons in a loop over the `gas_names` string array.